bulk-site-create-clone.sh

```diff
@@ -121,10 +121,17 @@ do
         echo -e "Cloning code, database and files from the $SOURCE to $DESTINATION. This may take a while...\n\n"
         terminus site:clone $SOURCE $DESTINATION --yes
     fi
+	# Add a tag to the site for the project
+    echo -e "Adding the tag $PROJECT_SLUG to the $DESTINATION_NAME site...\n"
+    	terminus tag:add $DESTINATION_NAME ${ORG_UUID} $PROJECT_SLUG
```

Comment:
@davidneedham this looks like a good change but `ORG_UUID` is not required. We should add a conditional here to make sure it is passed before adding the tag